parquet/spec.go

```diff
@@ -94,5 +115,8 @@ func (s *ParquetSpec) Validate() error {
 	if !slices.Contains(allowedRootRepetitions, s.RootRepetition) {
 		return fmt.Errorf("invalid rootRepetition: %s. Allowed values are %s", s.RootRepetition, strings.Join(allowedRootRepetitions, ", "))
 	}
+	if s.MaxRowGroupLength != nil && *s.MaxRowGroupLength <= 0 {
+		return fmt.Errorf("invalid: maxRowGroupLength: %v. Must be zero or positive", *s.MaxRowGroupLength)
```

Comment:
Is zero or a small number ok?

Reply:
Zero would be going back to the default of the arrow parquetwriter. Updated this check to be `< 0`